src/test/java/Homework17.java

```diff
@@ -0,0 +1,32 @@
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework17 extends BaseTest{
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+        String expectedSongAddedMessage = "Added 1 song into \"Test Pro Playlist.\"";
+
+        navigateToPage();
+        provideEmail("demo@class.com");
+       providePassword("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(2000);
+        searchSong("Pluto");
+        Thread.sleep(2000);
+        clickViewAllBtn();
+       selectFirstSongResult();
+       clickAddToBtn();
+        choosePlaylist();
+        Assert.assertEquals(getAddToPlaylistSuccessMessage(), expectedSongAddedMessage);
+    }
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)